From 031913b9dfd1ad26640d1dc39423c1444e1fe140 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 12 Jul 2019 13:55:24 +0200
Subject: [PATCH 013/152] workspace: Use Workspace prefix for WindowClone class

We have both Workspace's WindowClone and workspaceThumbnail's WindowClone,
so better to be clear about them using a class prefix.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/700
---
 js/ui/workspace.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/workspace.js b/js/ui/workspace.js
index f0951a63d..5459b8baf 100644
--- a/js/ui/workspace.js
+++ b/js/ui/workspace.js
@@ -104,7 +104,7 @@ var WindowClone = GObject.registerClass({
         'show-chrome': {},
         'size-changed': {}
     },
-}, class WindowClone extends St.Widget {
+}, class WorkspaceWindowClone extends St.Widget {
     _init(realWindow, workspace) {
         this.realWindow = realWindow;
         this.metaWindow = realWindow.meta_window;
-- 
2.23.0

